### ISSUE TITLE
Unable to build after recent changes with Feedback Email feature

### ISSUE THREAD

**Issue by wahibhaq** [codernaut/citizenconnect#17]
I fetched recent changes and now can't build because of recent changes regarding `SendEmail` functionality. 

I can see that 3 external libs were added in build.gradle file

```
    implementation files('/Development/Code for Pakistan/citizenconnect/Android App/libs/activation.jar')
    implementation files('/Development/Code for Pakistan/citizenconnect/Android App/libs/additionnal.jar')
    implementation files('/Development/Code for Pakistan/citizenconnect/Android App/libs/mail.jar')
```

But there were two problems with this approach:

1) They are absolute paths which means every contributor is bound to follow it which is not possible. It should have been like `implementation files('libs/mail.jar')`
2) The `libs` folder along with the jar files was not pushed to the repo or at least I don't see it which is the main reason I can't build it. 

---

Something unrelated to the fix but I can see that there is a feedback form where user can enter message for feedback/question and an email is sent to `shahzaib.shahid414@gmail.com`. I can also see that the code was used from this [tutorial](https://androidmads.blogspot.de/2016/06/java-mail-api-using-gmail-oauth-api-in.html).

Honestly, I don't understand the need for this hassle of creating separate utility classes and adding external libs which should be the max last resort and not so recommended. It could have been dealt in a very easy way with allowing user to send Email by only showing the email clients to choose and pre-filling the recipient, subject and even body field where user can further add his/her message and just click send. 

What are your thoughts? Was there a very critical reason for such feature to programmatically send feedback?

**Comment by codernaut** [comment]
Q: "Showing the email clients to choose and pre-filling the recipient, subject and even body field where user can further add his/her message and just click send." 

A: We need to add value and we will be auto generating most of emails so thats why we need our own client. We are verifying number also

Q:Adding external libs which should be the max last resort and not so recommended
A: Go ahead, turn them in to gradle dependencies they must be published in some repo. Will be obliged.

**Comment by wahibhaq** [comment]
should we close this issue? Was this fixed already?

**Comment by shahzaib414** [comment]
Yes , I moved sending email from App  to server. Will clean App code once
we tested App properly  i mean those libraries which we added for sending
Email in App

**Comment by wahibhaq** [comment]
That's even better. 

So app will send the feedback information to our backend and then backend will send it to the recepient?

**Comment by shahzaib414** [comment]
Yes
